mlflow/ml-package-versions.yml

```diff
@@ -469,3 +469,19 @@ transformers:
       ">= 0.0.0": ["datasets", "huggingface_hub", "torch", "torchvision", "tensorflow"]
     run: |
       pytest tests/transformers/test_transformers_model_export.py
+
+langchain:
+  package_info:
+    pip_release: "langchain"
+    install_dev: |
+      pip install git+https://github.com/hwchase17/langchain
+  models:
+    minimum: "0.0.139"
+    maximum: "0.0.139"
+    requirements:
+      ">= 0.0.0": ["pyspark", "transformers", "tensorflow"]
```

Comment:
Why do we need `pyspark` and `tensorflow` here, but not `langchain` itself and `openai`?

Reply:
As we spoke offline, tensorflow is needed by the hugging face transformer we are using in test. 
pyspark is needed to test spark_udf tests.
Also we don't directly use openai. we import openai from langchain in the test